073-decouple-ca-management-from-end-entity-certificate-management.md

```diff
@@ -0,0 +1,339 @@
+<!-- This template is provided as an example with sections you may wish to comment on with respect to your proposal. Add or remove sections as required to best articulate the proposal. -->
+
+# Decouple CA certificate management from end-entity certificate management
+
+This proposal aims to decouple the management of the cluster CA certificate from the rolling out of trust of the CA certificate to components in a Strimzi cluster.
+
+## Current situation
+
+To provide a secure, TLS-enabled setup by default when deploying Kafka clusters, Strimzi integrated its own CA operations into the Cluster Operator.
+The Cluster Operator accomplishes this by using `openssl` to generate self-signed root CA certificates and private keys which it then uses to directly sign end-entity (EE) certificates (i.e. intermediate certificates are not used).
+A _cluster CA_  and _clients CA_ are generated.
+These CAs are only used for Kafka clusters and a unique instance of each CA is used for each Kafka cluster.
+The cluster CA issues certificates for the following Strimzi components:
+* ZooKeeper nodes
+* Kafka nodes
+* User and Topic operators
+* Cruise Control
+* Kafka Exporter
+
+The clients CA issues certificates for user applications using the User Operator, or through another external mechanism chosen by the user.
+
+The cluster CA root certificate is added to the trust stores of Strimzi components.  This ensures that, for example, ZooKeeper nodes trust each other's certificates as well as the certificates of brokers, and brokers trust certificates issued to other brokers.
+
+#### Fig 1: Cluster CA certificate and component relationships
+
+![Cluster CA relationships](./images/073-cluster-ca.svg)
+
+> Fig 1: In the diagram, the red lines show trust.
+> For example, because the Kafka broker trusts the Cluster CA certificate,
+> and the Cluster CA certificate signed the ZK EE certificate
+> the broker will trust the ZK EE certificate presented by the ZK node
+> during TLS handshake.
+
+The cluster CA root certificate also needs to be trusted by Kafka clients connecting to the cluster so that clients trust the certificates presented by the brokers.
+The clients CA root certificate is added to the broker trust stores too, so that the brokers will trust certificates issued to Kafka client applications.
```

Comment:
maybe adding "... and presented by clients to brokers when the mTLS is enabled"?